ci/build_libtelio.py

```diff
@@ -243,9 +294,14 @@ def main() -> None:
 
 def exec_build(args):
     if args.moose:
-        if args.os in ["windows", "android"]:
+        if args.os in ["linux", "windows", "android"]:
             sys.path.append(f"{PROJECT_ROOT}/ci")
-            moose_utils.fetch_moose_dependencies(args.os, MOOSE_MAP[args.arch])
+
+            if (args.os, args.arch) == ("linux", "armv7"):
+                moose_utils.fetch_moose_dependencies(args.os, MOOSE_MAP["armv7hf"])
```

Comment:
:thinking: maybe there were some discussions and I am not aware of its outcomes, but it is not safe to assume `armv7hf` architecture when `armv7` is requested. The reverse would be safe, but in this direction. This may produce segfaults on non-hardfload CPUs.

Reply:
We call it `armv7` but underneath `armv7hf` is used.

Reply:
Should I set this arch to the more explicit `armv7hf` on release versions? This might break things on other places

Reply:
IMHO it would be preferable to set the arch itself to `armv7hf` in order to avoid this silent "gotcha" where `armv7` in fact means `armv7hf`. 

But I wonder how many things will be broken by attempting to change it :thinking:. If it's major effort to update it, then probably we can live with `armv7`, otherwise, it would be cool to avoid this gotcha.

Reply:
Done.